common/amount.h

```diff
@@ -102,6 +102,9 @@ bool amount_msat_less_eq_sat(struct amount_msat msat, struct amount_sat sat);
 WARN_UNUSED_RESULT bool amount_msat_to_u32(struct amount_msat msat,
 					   u32 *millisatoshis);
 
+/* Programatically initialize an msat from an u64. */
+void amount_msat_from_u64(struct amount_msat *msat, u64 millisatoshis);
```

Comment:
If I'm not mistaken the previous commit will fail to compile since it depends on these definitions. Please swap their order so we keep the history bisectable (each commit should compile and pass tests on its own).

Reply:
This is just a Github GUI glitch after I `git rebase -i` the commit order. As you can see on my local branch https://github.com/m-schmoock/lightning/commits/issue-2483-config-value the order is in fact correct.

```
* 5bb6756e (HEAD -> issue-2483-config-value, mschmoock/issue-2483-config-value) doc: commandline switch min-capacity-sat
* e7b650a1 test: opening_tiny_channel with min_capacity_sat
* a6df7938 feat: add min_capacity_sat config value and switch
* be0e5c97 common: add programatic initialization of msat by u64
* e1e10cce fix: openingd memory leaks when failing
```

Can you confirm this?

Reply:
@cdecker  There is this GitHub bug: https://help.github.com/en/articles/why-are-my-commits-in-the-wrong-order

Reply:
I will do this, so this can be closed: https://stackoverflow.com/a/44705243/884474